<?xml version="1.0"?>
<launch>
	<arg name="top" default="False"/>
	<arg name="bottom" default="False"/>
	<arg name="file" value="auto_exposure.py"/>
	<group if="$(arg top)">
		<node pkg="zeabus_vision" name="auto_exposure" type="$(arg file)" output="screen">
			<param name="imageWidth" value="1152" />
			<param name="imageHeight" value="874" />
			<param name="cameraPos" type="str" value="top" />
			<param name="cameraTopicLeft" type='str' value='/top/center/image_rect_color/compressed'/>
			<!-- <param name="cameraTopicRight" type='str' value='/top/center/image_rect_color/compressed'/> -->
			<param name="cameraNodeLeft" type='str' value='ueye_cam_nodelet_top_center/'/>
		</node>
	</group>
	<group if="$(arg bottom)">
		<node pkg="zeabus_vision" name="auto_exposure" type="$(arg file)" output="screen">
			<param name="imageWidth" value="1280" />
			<param name="imageHeight" value="768" />
			<param name="cameraPos" type="str" value="bottom" />
			<param name="cameraTopicLeft" type='str' value='/bottom/left/image_raw/compressed'/>
			<!-- <param name="cameraTopicRight" type='str' value='/bottom/right/image_raw/compressed'/> -->
			<param name="cameraNodeLeft" type='str' value='ueye_cam_nodelet_bottom_left/'/>
			<!-- <param name="cameraNodeRight" type='str' value='ueye_cam_nodelet_bottom_right/'/> -->
		</node>
	</group>
</launch>	